STM32Resources/install.sh

```diff
@@ -34,7 +60,12 @@ if ! type "CubeMX2Makefile" > /dev/null; then
     echo "ABS_PATH=\"\$(readlink -f \$1)\"" >> /usr/local/bin/CubeMX2Makefile
     echo "(cd /opt/CubeMX2Makefile && python CubeMX2Makefile.py \$ABS_PATH)" >> /usr/local/bin/CubeMX2Makefile
     chmod +x /usr/local/bin/CubeMX2Makefile
-    echo "export PATH=\$PATH:/opt/CubeMX2Makefile" >> ~/.profile
+    if [ "$DISTRO" = "Ubuntu" ]; then
+        BASH_PROFILE = "~./profile"
```

Comment:
Just set this once at the top of the file and keep reusing it.  No need for the same if statements.